src/app/components/LiveLabel/index.tsx

```diff
@@ -8,22 +8,39 @@ import styles from './index.styles';
 import Text from '../Text';
 
 interface LiveLabelProps {
-  ariaHidden?: boolean;
-  liveText?: string;
   offScreenText?: string;
   lang?: string;
   id?: string;
 }
 
 const LiveLabel = ({
-  ariaHidden = false,
-  liveText = 'LIVE',
-  offScreenText = '',
   lang = 'en-GB',
   id = '',
   children,
+  offScreenText,
 }: PropsWithChildren<LiveLabelProps>) => {
-  const { dir } = useContext(ServiceContext);
+  const { dir, translations } = useContext(ServiceContext);
+
+  const { liveLabel } = translations.media;
+
+  // As screenreaders mispronounce the word 'LIVE', we use visually hidden
+  // text to read 'Live' instead, which screenreaders pronounce correctly.
+
+  const liveLabelIsEnglish = liveLabel === 'LIVE';
+
+  let screenReaderText;
+  let ariaHidden = false;
+
+  if (liveLabelIsEnglish) {
+    ariaHidden = true;
+  }
```

Comment:
Let's add some comments to explain what's going on.



Should we add comments to this code to explain what is happening?

```suggestion
  // If the translated phrase for "LIVE" is English, we don't want this to be read out by the screenreader, as it is often pronounced incorrectly (rhyming with "give" instead of "hive"), hence why we set ariaHidden to true
  if (liveLabelIsEnglish) {
    ariaHidden = true;
  }
```

Reply:
Yeah comments are great. They save so much time in trying to work out or remember what is going on.